packages/react-components/react-color-picker-preview/library/src/components/ColorArea/ColorArea.test.tsx

```diff
@@ -9,10 +9,33 @@ describe('ColorArea', () => {
     displayName: 'ColorArea',
   });
 
-  // TODO add more tests here, and create visual regression tests in /apps/vr-tests
-
   it('renders a default state', () => {
-    const result = render(<ColorArea>Default ColorArea</ColorArea>);
-    expect(result.container).toMatchSnapshot();
+    const result = render(<ColorArea color="red" />);
+    expect(result.container).toMatchInlineSnapshot(`
+      <div>
+        <div
+          class="fui-ColorArea"
+          style="--fui-AreaX--progress: undefined%; --fui-AreaY--progress: undefined%; --fui-Area__thumb--color: transparent; --fui-Area--main-color: hsl(undefined, 100%, 50%);"
+        >
+          <input
+            aria-label="Color picker"
+            aria-valuetext="Saturation: undefined%, Value: undefined%"
```

Comment:
Is it expected to have values like `undefined%`?

Reply:
I'll update the tests. It's left from a11y testing